content/500-support/100-help-articles/500-comparing-columns-through-raw-queries.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Comparing columns of same table through raw queries'
+metaTitle: 'Comparing columns of same table through raw queries'
+metaDescription: 'Comparing column values through prisma's inbuilt raw query methods'
+---
+
+## Issue
+
+Comparing different columns from the same table is a common scenario that developers encounter. Some examples include comparing two numeric values in the same table or comparing two dates in a same table.
```

Comment:
Maybe add the link to the issue that tracks this as a proper feature of Prisma Client?

Reply:
Added the GitHub Issue Link - https://github.com/prisma/prisma/issues/5048